src/main/java/duke/Command.java

```diff
@@ -0,0 +1,5 @@
+package duke;
+
+public enum Command {
```

Comment:
The enumeration was well-defined. 
You might consider make the Command an abstract class as well, with method like `execute()` so you can have children classes and inherite from the parent class.